parsers/tex.c

```diff
@@ -35,7 +35,7 @@
 #define isType(token,t)		(bool) ((token)->type == (t))
 #define isKeyword(token,k)	(bool) ((token)->keyword == (k))
 #define isIdentChar(c) \
-	(isalpha (c) || isdigit (c) || (c) == '$' || \
+	(isalpha (c) || isdigit (c) || (((unsigned char) c) > 0x80) || (c) == '$' || \
```

Comment:
Does this work with `À`? I think that's the two bytes `0xC380` in utf-8, so it would fail this check for the second byte. So I think it should be ` >= 0x80`.

Reply:
Oops, you're totally right, just a typo on my part. Fixed -- and it even improved my test results further, as they used some 0x80 bytes.